build.gradle

```diff
@@ -6,12 +6,15 @@ buildscript {
     }
   }
   dependencies {
+    classpath group: 'com.cinnober.gradle', name: 'semver-git', version: '2.2.0'
```

Comment:
Nit: could you keep the style consistent? I don't have a strong opinion either way.